### PR TITLE
Automate RVA ID Validation

### PR DESCRIPTION
**Pull request by mzack5020**
## 🗣 Description
Removed dependency on the user providing a file with the proper format. Previously, the user must provide a file that began with the correctly formatted RVA ID. The RVA ID validation now uses a regular expression to validate the RVA ID within the data itself. 

This pull request resolves issue #12.

## 💭 Motivation and Context
#12 Requiring the user to provide a properly formatted filename is an unnecessary manual step that adds additional steps.

## 🧪 Testing
Tested the new RVA ID validation logic using my local Python environment. Additionally, @mcdonnnj tested using a complete deploy using the Terraform code for this project. 

## 🚥 Types of Changes
- [X] Breaking change (causes existing functionality to change)

## ✅ Checklist
- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: fdi/findings_data_import.py [hunk 1]
```diff
@@ -133,11 +134,6 @@ def import_data(
     temp_file_descriptor, temp_data_filepath = tempfile.mkstemp()
 
     try:
```
Comment (jsf9k) [inline fdi/findings_data_import.py:136]: A potential improvement is to wrap each piece of code that could throw an exception in its own `try` block, and catch only the exceptions that can be thrown in the corresponding `catch` block.  That way you can give a context-specific error or warning message when you catch the exception, and be more specific about which exceptions you are catching.

This doesn't need to hold up this PR, but it's an idea for a future improvement.

**Review by jsf9k — APPROVED** [review]
Approved, but please see my comment regarding a suggested future improvement.

File: fdi/findings_data_import.py [hunk 2]
```diff
@@ -199,10 +195,17 @@ def import_data(
                         finding[field_map_dict[field]] = finding[field]
                     finding.pop(field, None)
 
+            # Get RVA ID from last 4 digits
+            rvaId = re.search(r"(\d{4})(?:\.\d{1,})?$", finding["RVA ID"])
```
Comment (felddy) [inline fdi/findings_data_import.py]: _Code never lies, comments sometimes do._
–Ron Jeffries

This comment does not match the function of the regular expression that follows.
See: https://regexr.com/4p7pc

Please rewrite to include the information in the `README.md`

**Review by felddy — CHANGES_REQUESTED** [review]

**Review by felddy — APPROVED** [review]

File: fdi/findings_data_import.py (hunk 1 above)
Comment (dav3r) [reply fdi/findings_data_import.py:136]: @mzack5020 or @mcdonnnj - Can one of you please add a brief issue to this repo with @jsf9k's suggested improvement?

**Review by dav3r — APPROVED** [review]
Looks good gentlemen! 👍

File: fdi/findings_data_import.py (hunk 1 above)
Comment (mcdonnnj) [reply fdi/findings_data_import.py:136]: Please see #15 for future work.

**Pull request merged**